client/admin/retryableClient.go

```diff
@@ -114,16 +114,16 @@ func (c *retryableClient) CloseShard(
 	return resp, err
 }
 
-func (c *retryableClient) DescribeWorkflowExecution(
+func (c *retryableClient) DescribeMutableState(
 	ctx context.Context,
-	request *adminservice.DescribeWorkflowExecutionRequest,
+	request *adminservice.DescribeMutableStateRequest,
 	opts ...grpc.CallOption,
-) (*adminservice.DescribeWorkflowExecutionResponse, error) {
+) (*adminservice.DescribeMutableStateResponse, error) {
```

Comment:
breaking change?

Reply:
Yes. Unfortunately I didn't have time to fix this before release. But this API is used by admin CLI only, which is ok to break. I will outline this in next release notes (one `tctl admin` command works works only with corresponding version of server).